docs/edgeql/funcops/array.rst

```diff
@@ -32,10 +32,10 @@ Array
     * - :eql:func:`find`
       - Find the index of an element in the array.
 
-    * - :eql:func:`to_str`
+    * - :eql:func:`array_join`
       - Render an array to a string.
 
-    * - :eql:func:`to_array`
+    * - :eql:func:`str_split`
```

Comment:
This one should be moved to https://github.com/edgedb/edgedb/blob/master/docs/edgeql/funcops/string.rst